.github/actions/gam-canary-rollback/canary-probe.mjs

```
@@ -0,0 +1,156 @@
+if (expectHeader) {
+  const i = expectHeader.indexOf(':');
+  if (i < 0) refuseConfig('EXPECT_HEADER must be "Name: value-substr"');
+  hdrName = expectHeader.slice(0, i).trim().toLowerCase();
+  hdrWant = expectHeader.slice(i + 1).trim();
+}
+
+async function probeOnce() {
+  const ctrl = new AbortController();
+  const t = setTimeout(() => ctrl.abort(), timeoutMs);
+  try {
+    const res = await fetch(url, {
```

ESLint

[error] 'fetch' is not defined.

(no-undef)